meetings/2023-10-11.md

```diff
@@ -67,24 +125,55 @@ URL: https://github.com/solid/specification/discussions/581
 URL: https://github.com/solid/specification/discussions/576
 
 * SC: Just a stub for now.
+* SC: I don't want it to focus on punishment, more about having a healthy culture.
 
 
 
-### [New Work Item Proposal] Solid-PREP
-URL: https://github.com/solid/specification/issues/580
+### Solid CG Chair Election Procedure
+URL: https://github.com/solid/specification/discussions/582
 
-* SC: Proposed by RG.
+* SC: Summary..
+* SC: Everyone should review it, I will unlock it as soon as I get it reviewed by W3C staff.
+* SC: They are on-board and interested to see how it is going to work.
+* SC: We need volunteers to review eligible participants list, based on charter and affiliations.
+* SC: W3C can't act as 3rd party, they don't have resources to do that. They will help us to anonymise the results and use W3C tool to process the results.
+* SC: We need to agree on when do we start.
+* SC: We also need volunteers to compute the results.
+* JZ: I don't have any problem in general. I agree with Tim that people should be encouraged to give their vision of how they want to chair. We shouldn't suggest how one should chair the CG.
+* VB: I'm happy to help with counting and processing. The self-review is meant as an exercise for oneself and not a list of question that one has to answer.
+* SC: W3C has a document on chair role which is on top of that discussion. It has some guidelines of what is expected from chairs.
+* SC: We also hope for 3 chairs which will bring diverse perspectives.
+* SC: We need at least one more volunteer besides Virginia
+* SC: Should we set date for next Wednesday for snapshotting eligible participants? For voting and chair nominations.
+* VB: We can go with 1 week
+* HZ: +1
+* PAC: +1
+* HZ: I can also volunteer
+* TT: To review participants list or to chair?
+* SC: To review the list and run that open STV tool from W3C.
+* eP: I can also volunteer
 
 
-### Solid Protocol Version 0.11.0
-URL: https://github.com/solid/specification/milestone/7
 
-* SC: Let's make sure to add any missing issues/PRs that can reasonably make it into this release. The ED includes class 3 and higher changes, and some in the pipeline. See [Solid Protocol ED Changelog](https://solidproject.org/ED/protocol#changelog).
-* SC: Unless there is new information to discuss, I suggest we keep this topic short and only for awareness.
+### [New Work Item Proposal] Solid-PREP
+URL: https://github.com/solid/specification/issues/580
+
+* SC: Proposed by RG. Please ping in the issue to co-own so it'll be a work item.
+* RG: I have mentioned everything in the issue.
+* RG: I don't expect it a lot of work, there is already an open issue for that.
+* eP: Issue relates to storage operations. There is another issue in solid/specification (which is unresolved). We could move to the core perhaps in the WG. That seems like a dependency in these kinds of activities. There may be complications on how this could be resolved.
```

Comment:
```suggestion
* RG: I don't expect it to be a lot of work; there is already an open issue for that.
* eP: Issue relates to storage operations. There is another issue in solid/specification (which is unresolved). We could move to the core, perhaps in the WG. That seems like a dependency in these kinds of activities. There may be complications on how this could be resolved.
```